spec/lib/presenters/show_presenter_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+require "presenters/show_presenter"
+
+RSpec.describe ShowPresenter do
+  describe "#page_title" do
+    it "is the resource's key attribute" do
+      customer = build(:customer, name: "Worf")
+      presenter = ShowPresenter.new(CustomerDashboard.new, customer)
+
+      expect(presenter.page_title).to eq("Worf")
+    end
+  end
+
+  describe "#attribute_names" do
+    it "uses attribute names from the dashboard" do
+      customer = build(:customer, name: "Worf")
+      presenter = ShowPresenter.new(CustomerDashboard.new, customer)
```

Comment:
`CustomerDashboard` is a mystery guest in these specs. It would be great to define a dashboard (`double`?) within the spec, so we know what its contents are.